### PR TITLE
new blog of kubesphere-nocalhost

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao@kubesphere.io>

### CODE REVIEW

File: content/zh/blogs/kubesphere-nocalhost.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
+---
+title: 'KubeSphere 结合 Nocalhost 提供更强大的云原生开发环境'
```
Comment (yangchuansheng) [inline content/zh/blogs/kubesphere-nocalhost.md:2]: 在 Kubernetes 中部署云原生开发工具 Nocalhost

File: content/zh/blogs/kubesphere-nocalhost.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+tag: 'KubeSphere,Nocalhost'
+keyword: 'Kubernetes, KubeSphere, Nocalhost '
+description: '本文将介绍如何在 KubeSphere 中快速部署 Nocalhost Server，用于提供一个帮助研发团队统一管理 Nocalhost 应用部署的管理平台。'
```
Comment (yangchuansheng) [inline content/zh/blogs/kubesphere-nocalhost.md:5]: change 'KubeSphere' to 'Kubernetes'

File: content/zh/blogs/kubesphere-nocalhost.md (hunk 1 above)
Comment (faweizhao26) [reply content/zh/blogs/kubesphere-nocalhost.md]: Done.

File: content/zh/blogs/kubesphere-nocalhost.md (hunk 2 above)
Comment (faweizhao26) [reply content/zh/blogs/kubesphere-nocalhost.md]: Done.

**Pull request merged**